plugins/native/module/vtkF3DSPZReader.cxx

```diff
@@ -280,7 +316,7 @@ int vtkF3DSPZReader::RequestData(
   const Header* header = reinterpret_cast<Header*>(uncompressed.data());
 
   assert(header->magic == 0x5053474e);
-  assert(header->version == 2);
+  assert(header->version >= 2 && header->version <= 3);
```

Comment:
how about proper magic and version check instead ? assert will only fail in debug, which is not helpful for our users with somehow incorrectly formatted and wrongly versionned files.